flow/visualize/visualizer_rllib.py

```diff
@@ -110,6 +110,7 @@
     module = __import__("flow.envs", fromlist=[flow_params["env_name"]])
     env_class = getattr(module, flow_params["env_name"])
     env_params = flow_params['env']
+    env_params.evaluate = True
```

Comment:
should we make this an input argument?

Reply:
Yeah, we should. It should probably default to false

Reply:
Otherwise, people will be confused why their trained reward is off